openerp/tools/translate.py

```diff
@@ -329,6 +329,8 @@ def next(self):
                 elif line.startswith('#,') and (line[2:].strip() == 'fuzzy'):
                     fuzzy = True
                 line = self.lines.pop(0).strip()
+            if 0 == len(self.lines):
```

Comment:
I wrote that to be consistent with lines 305 and 339.  Should I change these other lines too?